### ISSUE TITLE
drag and drop actions (changing order of actions)

### ISSUE THREAD

**Issue by ducin** [zalmoxisus/redux-devtools-extension#622]
Is it possible to drag and drop an action, i.e. move an action as if it happened earlier, or later, than it actually happened?

What I'm trying to achieve is to test my reducers with the same set of actions but in different order action combinations. What I can do now is to manually re-dispatch these actions in different orders, separately for all combinations. But maybe it's possible to drag and drop them and I'm just missing that feature?

**Comment by zalmoxisus** [comment]
It was implemented in [`2.12.0`](https://github.com/zalmoxisus/redux-devtools-extension/releases/tag/v2.12.0), but got broken shortly after, for more than an year for sure. Nobody reported so far, so I thought it's a fancy useless feature. Maybe some incompatibilities of [`react-dragula`](https://github.com/bevacqua/react-dragula) and newer versions of react, I'll look into it.

Here's how it used to work:
![f5094dea-d2b3-11e6-8e74-899df8a0d1a4](https://user-images.githubusercontent.com/7957859/50742926-1ad30f00-121a-11e9-99ee-b8e1d354bfea.gif)

**Comment by ducin** [comment]
YES! I was using it in my old project, not got back to it. And had a hard time trying to remind if the feature was there or if I've got morbid imagination ;)
